db/migrate/20220527125023_create_userrs.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUserrs < ActiveRecord::Migration[7.0]
+  def change
+    create_table :userrs do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :postts_counter
+
+      t.timestamps
```

Comment:
- [ ] Kindly fix your tables migration, in your code you are referencing the `user` table but the created is `userrs` and it stops the migration to be complete.
![Screen Shot 2022-07-14 at 04 23 42](https://user-images.githubusercontent.com/28307960/178884473-66fc11f5-0715-4365-8f02-adf3f9fc3d5b.png)